yb-voyager/cmd/importSchemaYugabyteDB.go

```diff
@@ -141,25 +154,48 @@ func executeSqlFile(file string, objType string, skipFn func(string, string) boo
 			log.Infof("Skipping DDL: %s", sqlInfo.stmt)
 			continue
 		}
+		if isSessionVariable(sqlInfo.stmt) && !skip {
+			sessionVariables = append(sessionVariables, sqlInfo)
+			err = applySessionVariable(conn, sqlInfo)
+			if err != nil {
+				return err
+			}
+			continue
+		}
 
-		err = executeSqlStmtWithRetries(&conn, sqlInfo, objType)
+		err = executeSqlStmtWithRetries(&conn, sqlInfo, objType, sessionVariables)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
+func isSessionVariable(stmt string) bool {
+	parseTree, err := queryparser.Parse(stmt)
+	if err != nil {
+		return false
```

Comment:
return error if it fails; this will be a unexpected scenario and should be ok to fail.